File: CodeforcesPy/71A.py
# Enter test cases
testCases = int(input())

while(testCases):
    word = input()
    length = len(word)
    if length > 10 :
        print(word[0]+str(length-2)+word[length-1])
    else: 
        print(word)
    testCases-=1

File: Codeforces/Div 2A/71A.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
    int testCases;
    cin>>testCases;

    while(testCases){
        string word;
        cin>>word;

        if ( word.length() > 10){
            cout<<word[0]<<word.length()-2<<word[word.length()-1]<<endl;
        }else {
            cout<<word<<endl;
        }


        testCases--;
    }

    return 0;
}